examples/2-rdfparser/exampleRDFParser.go

```diff
@@ -24,18 +24,20 @@ func main() {
 	xmlReader, err := xmlreader.XMLReaderFromFilePath(filePath)
 	if err != nil {
 		// error reading the rdf file.
-		fmt.Println(err.Error())
+		fmt.Printf("Error reading the rdf file %v: %v", filePath, err)
```

Comment:
It is a general convention that the error string must not be capitalized in golang. Other parts of the codebase has followed it. Would be better if it is consistent.

Reply:
I made the necessary changes